RELEASE_NOTES.rst

```diff
@@ -175,8 +175,7 @@ are added for existing bears.
 
 New bears:
 -  ``AutoPrefixBear`` (Add vendor prefixes automatically in CSS)
--  ``ClangComplexityBear`` (Calculates cyclomatic complexity of each function
```

Comment:
> Nothing else.

Not true ;-)

You moved the indented text to the previous line.  That makes it a very long line, which isnt acceptable.

I think the required fix is to remove one space from the indent. on the original line 179, so it is aligned with the other text.